src/services/LibraryService.ts

```diff
@@ -0,0 +1,29 @@
+import { RequestArgs } from '@projecttacoma/node-fhir-server-core';
+import { findResourceById } from '../db/DBOperations';
+import { Service } from '../types/service';
+import { ResourceNotFoundError } from '../util/errorUtils';
+
+/*
+ * Example service that one would implement for the `Library` resource
+ * See the Service interface for all possible functions
+ */
```

Comment:
Guess this is no longer an example now that we have something huh?

Reply:
😮